sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -1708,17 +1729,6 @@ class CastSuite extends CastSuiteBase {
     }
   }
 
-  test("SPARK-35698: cast timestamp without time zone to string") {
```

Comment:
The test case was put in CastSuite. This PR moves it to CastSuiteBase so that test cases of ANSICast will be covered.